CONTRIBUTING.md

```diff
@@ -229,6 +229,12 @@ This involves the following steps:
 
    Provide a meaningful description of your changes in the textbox, press the button "Create pull request" and wait for somebody to merge your changes in (or get back to you with comments/requests). Note that subsequent changes (in response to discussions/requests, say) can simply be submitted to the same branch (repeating everything from step 3 above); they will automatically be added/included to the same pull request.
 
+   Note that when you return to the branch's GitHub page you won't see the lovely green ![](doc/README/compare_and_pull_request_button.png) button any more (presumably because a pull request is already pending). To add the newly made changes to the pull request, click on the "Contribute" button, next to the "Fetch upstream" one
```

Comment:
Can we remove the image embedded in the middle of the sentence? It looks rather messy at the moment. We could get away with just saying '...green "Compare & pull request" button...' instead of '...green \!\[\]\(doc/README/compare_and_pull_request_button.png\) button...'.

Reply:
Done! (Lovely trial run through the updating a pending pull request -- I'm no longer scared!)